src/js/route.js

```diff
@@ -15,10 +12,15 @@ Router.prototype.toString = function() {
 };
 
 Router.prototype.constructDomain = function() {
-    if (this.absolute)
+    if(this.name === undefined) {
```

Comment:
Yeah I actually saw this after I merged. That was the reason for my wanting to revisit the tests before I tag a new release 😄 #61